opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/ForceRescanScan.java

```diff
@@ -92,8 +92,9 @@ protected void scanExistingNode(final BatchTask phase) {
             OnmsIpInterface iface = m_provisionService.getPrimaryInterfaceForNode(node);
             if (iface == null) { // NMS-6380, a discovered node added with wrong SNMP settings doesn't have a primary interface yet.
                 iface = node.getIpInterfaces().isEmpty() ? null : node.getIpInterfaces().iterator().next();
-            } else {
-                LOG.info("The node with ID OP does not have a primary interface", m_nodeId);
+                if (iface != null) {
```

Comment:
This is just a very nit comment, but why change the else to an if, if you only want to adjust the log message?

Reply:
It wasn't just changed from "else" to "if"; I also moved the log statement inside the "if (iface == null) { }" block.
(The "OP" typo was a less critical issue, and I probably wouldn't have bothered to do a pull request for that alone.)

The log message used to trigger when iface actually existed, which was the opposite of what I presume the original author intended. Now it will correctly trigger *if* there is an interface, but it's not the primary SNMP interface, which is more aligned with the actual log message.